src/dolphin/atmosphere/ionosphere.py

```diff
@@ -0,0 +1,391 @@
+from __future__ import annotations
+
+import datetime
+import re
+from os import fspath
+from pathlib import Path
+from typing import Mapping, Sequence
+
+import numpy as np
+import opera_utils as oput
+from rasterio.crs import CRS
+from rasterio.warp import transform_bounds
+from scipy import interpolate
+
+from dolphin import io
+from dolphin._dates import _format_date_pair, get_dates
```

Comment:
```suggestion
from opera_utils._dates import get_dates
from dolphin.io import _format_date_pair
```
(sorry for the rearrangement)